modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -4,77 +4,203 @@ import {Meta} from '@storybook/addon-docs';
 
 # Canvas Kit 8.0 Upgrade Guide
 
-This guide contains breaking changes in Canvas Kit v8. Please
+This guide contains an overview of the changes in Canvas Kit v8. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
 - [Codemod](#codemod)
-- [Components](#components)
-  - [Component Deprecations](#component-deprecations)
+- [General Updates](#general-updates)
+  - [IE11 No Longer Supported](#ie11-no-longer-supported)
+  - [Removal of Default Exports](#removal-of-default-exports)
```

Comment:
We may want to include a section on the upgrade to React 18 under `General Updates` even though we still support React 16.14 and 17.X in v8. Thoughts?

Reply:
I think a link to the discussion would be great. I'll leave a placeholder `React 18 Upgrade` section with a reminder to fill the section in a follow-up PR.